Week3/README.md

```diff
@@ -10,26 +10,32 @@ In week three we will discuss the following topics:
 • Functions
 ```
 
+
+> :boom: IMPORTANT: The concepts of these weeks are extremely important, and require very careful reading. 
```

Comment:
`IMPORTANT` => repeats on => `are extremely important`

Reply:
I don't understand this

Reply:
it it ready to merge now?